pkg/object/azure.go

```diff
@@ -106,7 +106,7 @@ func (b *wasb) Put(key string, data io.Reader) error {
 	return err
 }
 
-func (b *wasb) Copy(dst, src string) error {
+func (b *wasb) Copy(dst, src, srcBucket string) error {
```

Comment:
It will be wrong if the `srcBucket` is different, we could add another method to support, otherwise we should return  error if it's not supported.